### DISCUSSION TITLE
it looks like this theme supports asset cache busting using unixtime?

### DISCUSSION THREAD

**Discussion by natemarks** [McShelby/hugo-theme-relearn#367] [Q&A] [answered]
I just saw the setting, but apparently it's enabled by default. Not sure how to validate that it's working. I kind of expect to see files with unix time stamps appended to names?  I'm mostly interested in testing with an S3/cloudfront deployment

 Thanks!

**Comment by McShelby** [comment, marked as answer]
The timestamp will be appended to the query string of an asset URL. You can see this in the network tab of your browser dev tools.

**Reply by natemarks** [reply]
thanks!
